bundle/manifests/authorino-operator.clusterserviceversion.yaml

```diff
@@ -283,6 +283,12 @@ spec:
                 resources: {}
                 securityContext:
                   allowPrivilegeEscalation: false
+                  capabilities:
+                    drop:
+                    - ALL
```

Comment:
Just wondering... We already run as non-root as escalation of privilege is not allowed. Isn't `capabilities.drop: ["ALL"]` redundant in this case?

```sh
$ docker run --cap-add SYS_ADMIN --user 1001 --entrypoint /bin/grep quay.io/kuadrant/authorino-operator:latest -- Cap /proc/self/status
CapInh:	0000000000000000
CapPrm:	0000000000000000
CapEff:	0000000000000000
CapBnd:	00000000a82425fb
CapAmb:	0000000000000000

$ docker run --cap-add SYS_ADMIN --user 1001 --entrypoint /bin/grep <image-based-on-this-branch> -- Cap /proc/self/status
CapInh:	0000000000000000
CapPrm:	0000000000000000
CapEff:	0000000000000000
CapBnd:	00000000a82425fb
CapAmb:	0000000000000000
```

Reply:
Yes, I agree. when I was deploying the operator, I got this warning

```
Warning: would violate PodSecurity "restricted:v1.24": unrestricted capabilities (container "manager" must set securityContext.capabilities.drop=["ALL"]), seccompProfile (pod or container "manager" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost") 
```
I think "dropping all capabilities" will only reduce any further risk of remote code execution, which seems more like a good-to-have thing. run as non-root already handle it. When I added both recommendations, it suppressed the warning.

Reply:
> When I added both recommendations, it suppressed the warning.

Good enough for me.

Thanks, @akoserwal!